src/Amazon.Extensions.S3.Encryption.csproj

```diff
@@ -15,43 +15,27 @@
         <PackageIcon>icon.png</PackageIcon>
         <RepositoryUrl>https://github.com/aws/amazon-s3-encryption-client-dotnet/</RepositoryUrl>
         <Company>Amazon Web Services</Company>
-        <AssemblyVersion>2.1.2</AssemblyVersion>
-        <FileVersion>2.1.2</FileVersion>
-
         <SignAssembly>true</SignAssembly>
         <AssemblyOriginatorKeyFile>..\public.snk</AssemblyOriginatorKeyFile>
-
-        <!-- workaround per https://github.com/Microsoft/msbuild/issues/1333 -->
-        <FrameworkPathOverride Condition="'$(TargetFramework)' == 'net35'">C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v3.5\Profile\Client</FrameworkPathOverride>
-
-        <!-- workaround per https://github.com/dotnet/msbuild/issues/5985 -->
-        <AutomaticallyUseReferenceAssemblyPackages Condition=" '$(TargetFramework)' == 'net35' ">false</AutomaticallyUseReferenceAssemblyPackages>
+        <PackageReadmeFile>README.md</PackageReadmeFile>
```

Comment:
Are we able to enable treat warnings as errors?

Reply:
Done